Websites/webkit.org/demos/webgpu/scripts/two-cubes.js

```diff
@@ -182,7 +181,8 @@ async function helloCube() {
         fragment: fragmentStageDescriptor,
         primitive: {
             topology: "triangle-list",
-            cullMode: "back"
+            cullMode: "back",
+            unclippedDepth: true
```

Comment:
Don't we have to enable the feature at device creation time?

Reply:
Yes that is correct. I will fix that